index.js

```diff
@@ -458,6 +458,8 @@ var characterMap = {
 	"Й":"И",
 	"ё":"е",
 	"Ё":"Е",
+	"Ể": "E",
+	"Ễ": "E",
```

Comment:
Unless I'm missing something, I see both of those in `master` already:

https://github.com/tyxla/remove-accents/blob/master/index.js#L37-L38

Are you sure we really need them?

Reply:
Ah probably not then. I just went through open PRs, but didn’t always double check whether they were obsolete. Feel free to close it. :)